spec/features/loan_index_spec.rb

```diff
@@ -1,13 +1,14 @@
 require 'rails_helper'
 
-feature 'visit home page' do
+feature 'visit loan index page' do
   before { pending 're-implement in new project' }
   before { @loans = create_list(:loan, 3, :active) }
-  context 'on the home page' do
-    before { visit root_path }
+  context 'on the loan index page' do
+    before { visit loans_path }
 
     it 'shows active loans' do
-      active_loans = @loans.select{ |loan| loan.status == I18n.t(:loan_active) }
+      active_loans = @loans.select{ |loan| loan.status == 'active' }
+      expect(active_loans).to be_present
```

Comment:
`be_present` means it responds `true` to the method `.present?`, I tossed that in there so it would fail in cases where it was empty, because `.each` will just iterate gladly over an empty collection without an error.